source/usage-examples/insertOne.txt

```diff
@@ -48,7 +48,7 @@ with the inserted document ``ObjectId`` in the value field:
 For additional information on the classes and methods mentioned on this
 page, see the following API documentation:
 
-- :java-sync-api:`insertOne <com/mongodb/client/MongoCollection.html#insertOne>`
+- :java-sync-api:`insertOne <com/mongodb/client/MongoCollection.html#insertOne(com.mongodb.client.ClientSession,TDocument)>`
```

Comment:
Do we want to consistently link to the overload that takes a ClientSession?  That's not the most common usage, unless we're aggressively pushing transactions.

Reply:
Whoops, chose that one by accident. Will fix.